amato/umbrel-app.yml

```diff
@@ -0,0 +1,21 @@
+manifestVersion: 1
+id: amato
+category: productivity
+name: Amato Timer
+version: "0.0.1.1"
+tagline: Simple work timer.
+description: >-
+  A simple timer with default 25 minute increments to track your work.
+releaseNotes: ""
+developer: Bitparasol
+website: http://bitparasol.com
+port: 3838
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
+submitter: ziadgit
```

Comment:
If you are the developer of this app, then we should make this field and the `developer` field have the same info so that users know that the developer of the app is also the submitter.